modules/products/bitbucket/helm.tf

```diff
@@ -61,7 +61,6 @@ resource "helm_release" "bitbucket" {
             create           = true
             storageClassName = ""
           }
-          subPath = "${local.product_name}-${random_string.random.result}"
```

Comment:
We don't use `subpath` anymore as this is useful for EFS that is shared across products but not necessarily for the NFS scenario.